kubectl-plugin/src/main.rs

```diff
@@ -8,8 +8,10 @@ mod resources;
 mod rest_wrapper;
 
 use crate::{
-    operations::{Get, List, Scale},
-    resources::{node, pool, utils, volume, GetResources, ScaleResources},
+    operations::{Get, List, ReplicaTopology, Scale},
```

Comment:
**[low-priority]**:- https://github.com/spf13/cobra#concepts, I have been using this and it says CLI commands are VERBS. Not sure about other best practices though.

Reply:
That's a good point, thanks. Maybe it can be `GetReplicas` instead.

Reply:
I have changed this as you suggested.